### ISSUE TITLE
Auto-capture video when specific window / process opened?

### ISSUE THREAD

**Issue by vinushkah** [ShareX/ShareX#3325]
Is it possible to have Share-X begin capturing video and / or screenshots targeting a specific window when that window is opened or brought to the foreground. 

I am wanting to auto-capture support sessions via VNC / TeamViewer so as soon as the client's screen window opens on my screen, ShareX begins recording.

**Comment by matthewburnett** [comment]
No, but you can use the hotkeys to quickly start recording without having to open sharex window.
